README.md

```diff
@@ -10,6 +10,8 @@ kube-applier runs as a Pod in your cluster and watches the [Git repo](#mounting-
 
 Whenever a new commit to the repo occurs, or at a [specified interval](#run-interval), kube-applier performs a "run", issuing [kubectl apply](https://kubernetes.io/docs/user-guide/kubectl/v1.6/#apply) commands with pruning at namespace level. The convention is that level 1 subdirs of REPO_PATH represent k8s namespaces: the name of the dir is the same as the namespace and the dir contains manifests for the given namespace.
 
+kube-applier will try to fetch a Service Account under each namespace (as defined by the environment variable `SERVICE_ACCOUNT_NAME`) and will use it to run kubectl commands. It will error for all namespaces that do not contain a matching SA.
```

Comment:
`ACTING_SERVICE_ACCOUNT_NAME`?

Reply:
Hm, I suppose this disambiguates it from the SA of the actual kube-applier service. Although I think the distinction is clear enough without it.

I think I prefer the sound of `ASSUMED_SERVICE_ACCOUNT_NAME`, but I don't know if this is an accurate description of what is actually happening.

Reply:
Happy for it to be something else, but if you have an application, and it has a flag called "service account name" - my first thought would be - the service account the application runs with, not the SA it gets from other namespaces to apply them.

Reply:
Changed.